### DISCUSSION TITLE
What to study to start contributing to kernel

### DISCUSSION THREAD

**Discussion by rathod-sahaab** [hannobraun/fornjot#2421] [General]
Apart from Rust and libraries (programming aspects) what to study to acquire domain specific knowledge like things related Computational Geometry

I have programming, maths and physics experience but DSL is lacking if you can recommend books/topics to study that would be great

**Comment by hannobraun** [comment]
Hey @rathod-sahaab, thank you for your interest. I fear I don't have a good answer. I got my own knowledge over many years, from reading many different things, byt mostly my own experimentation. And I don't feel I'm particularly good at any of it.

I think I got a lot out of Boundary Representation Modelling Techniques by Ian Stroud early on. Maybe that is a good place to start.

**Reply by rathod-sahaab** [reply]
Thanks!! Will look into it
